R/calculate_company_alignment_metric.R

```diff
@@ -354,54 +358,81 @@ calculate_company_aggregate_alignment_sda <- function(data,
     scenario = scenario
   )
 
+  # params
   start_year <- min(data$year, na.rm = TRUE)
+  # standard forward looking PACTA time frame
+  time_frame <- 5
```

Comment:
also here it's used again in a different function, which suggests
1. this will be difficult to get right if 5 ever changes to something else
2. this should maybe be a package level function/value/option that's available universally throughout the package

(these are just thoughts though, not saying you have to do this)

Reply:
I was wondering if I should surface it as an argument. I guess there is no harm in doing that for the sake of adjusting it later if necessary

Reply:
@cjyetman updated